docs/requirements.txt

```diff
@@ -0,0 +1,6 @@
+myst-parser>=2.0.0
+pydata-sphinx-theme>=0.15.2
```

Comment:
```suggestion
pydata-sphinx-theme==0.14.4
```

We might want to downgrade to this version to get over the [search results summaries not shown issue](https://github.com/pydata/pydata-sphinx-theme/issues/1676) until their next version release.

Before:
![image](https://github.com/y-scope/clp/assets/43196707/1f4b43d0-54f0-4f34-ada5-1383f497abaa)

After downgrade:
![image](https://github.com/y-scope/clp/assets/43196707/d1f8d753-7dbf-48ce-8c70-d316e9d275bf)